src/cmd-cloud-prune

```diff
@@ -0,0 +1,303 @@
+                if buildDate < refDate:
+                    for arch in build["arches"]:
+                        meta_url = f"/builds/{build_id}/{arch}/meta.json"
+                        meta_json = get_json_from_s3(bucket, prefix + meta_url)
+                        if meta_json is None:
+                            print(f"Failed to get meta.json for {build_id} for {arch}")
+                            return
+                        images = {
+                            "amis": meta_json.get("amis") or [],
+                            "gcp": meta_json.get("gcp") or [],
+                        }
+                        currentBuild = Build(
+                            id=build_id,
+                            timestamp=timestamp,
+                            images=images,
+                            arch=arch,
+                        )
+                        match action:
+                            case "cloud-uploads":
+                                # Prunes only AWS and GCP at the moment
+                                delete_cloud_resources(currentBuild, gcp_cloud_config, args.dry_run)
+                                if not args.dry_run:
+                                    build.setdefault("policy-cleanup", []).append(action)
+                                    buildJsonData["builds"][index] = build
+                            # Not implemented yet
+                            case "build":
+                                print(f"Deleting key {prefix}{build.id} from bucket {bucket}")
+                                # Delete the build's directory in S3
+                                # S3().delete_object(args.bucket, f"{args.prefix}{str(currentBuild.id)}")
+                else:
+                    break
+        if not args.dry_run:
+            with open("builds/builds.json", "w") as json_file:
+                json_file.write(json.dumps(buildJsonData))
+            # This copies the local builds.json and updates the S3 bucket version.
+            s3_copy(s3_client, BUILDFILES['list'], bucket, f'{prefix}/builds/builds.json',
+                    CACHE_MAX_AGE_METADATA, args.acl, extra_args={},
+                    dry_run=args.dry_run)
+
+
+def get_json_from_s3(bucket, key):
+    # Create an S3 client
+    s3 = boto3.client("s3")
+
+    try:
+        # Fetch the JSON file from S3
+        response = s3.get_object(Bucket=bucket, Key=key)
+
+        # Read the content of the file
+        content = response["Body"].read().decode("utf-8")
+
+        # Parse the JSON content
+        json_content = json.loads(content)
+
+        return json_content
+
+    except Exception as e:
+        print(f"Error fetching the JSON file from S3 {bucket}/{key}: {e}")
+        return None
+
+
+def parse_args():
+    parser = argparse.ArgumentParser(prog="coreos-assembler cloud-prune")
+    parser.add_argument("--policy", default='./policy.yaml', type=str,
```

Comment:
Not sure this should have a default. To start, let's just always require it (use `required=True`).